ci/prow/config_knative.yaml

```diff
@@ -79,6 +79,12 @@ presubmits:
       args:
         - "--run-test"
         - "./test/e2e-tests.sh --kourier-version stable"
+    - custom-test: ambassador-latest
+      always_run: false
+      optional: true
+      args:
+        - "--run-test"
+        - "./test/e2e-tests.sh --ambassador-version 0.86.1"
```

Comment:
Use "latest" instead of 0.86.1, here and below. Currently e2e-tests.sh will ignore the version and install the one hardcoded in the code (so this can generate confusion). Also, you'll have two places to change the version number in the future.

Reply:
Makes sense, fixed in separate commit. Thanks!